scripts/timer.py

```diff
@@ -0,0 +1,54 @@
+import tkinter as tk
+import time
+import threading
+import winsound
+
+root = tk.Tk()
+root.title("Timer")
+
+error_label = tk.Label(root, text="Please enter a valid number", font=("Times",12,), fg="#f00")
+entry_label = tk.Label(root, text="Enter timer duration (seconds):", font=("Times",48))
+entry_label.grid(row=0, column=0, pady=10)
+entry = tk.Entry(root)
+entry.grid(row=1, column=0, pady=10)
+
+timer_label = tk.Label(root, text="", font=("Times",48))
+
+def display_timer(seconds):
```

Comment:
Trivial: Can we add docstrings to functions to document what happens and what they expect as input (if any) and return as output(if any)?
For example:
```
def display_timer(seconds):
    """ Display the timer.
         args: seconds (int)
    """
```

Reply:
Is the new commits docstrings enough or do i need to go into more detail?

Reply:
It is perfect, thank you so much for adding them!